### PR TITLE
Added sparse matrix

### LINKED ISSUE #333: Sparse Matrix Representation
I would like to implement Sparse Matrix Representation using Linked list in C++ as part of GSSoC'21

### PR DESCRIPTION
**Pull request by aish2002**
Fixes #333 
Added sparse matrix in C++
@OjusWiZard , @tarun26091999 kindly review the PR

### CODE REVIEW

**Comment by aish2002** [comment]
@OjusWiZard kindly review the PR

File: Code/C++/Sparse_Matrix.cpp [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                new_node->row = i;
+                new_node->column = j;
+                new_node->data = mat[i][j];
+                new_node->next = NULL;
+                //If the list is empty insert as first node
+                if (head == NULL)
+                    head = new_node;
+                else
+                {
+                    node *ptr = head;
```
Comment (krishna-NIT) [inline Code/C++/Sparse_Matrix.cpp:47]: add space for *

Comment (krishna-NIT) [reply Code/C++/Sparse_Matrix.cpp:47]: repeat the same everywhere

**Comment by krishna-NIT** [comment]
Update the Readme which is present at the location ( of the logic/ concept used in question) , add this question this at Last of this Readme and arrange these question in alphabetical Order

**Comment by krishna-NIT** [comment]
> Update the Readme which is present at the location ( of the logic/ concept used in question) , add this question this at Last of this Readme and arrange these question in alphabetical Order

@aish2002

**Pull request closed without merging**